package.json

```diff
@@ -3,10 +3,6 @@
   "version": "1.8.0",
   "description": "Heroku CLI plugin for Heroku CI",
   "main": "index.js",
-  "engines": {
-    "node": ">=6.8.1 <8.0.0",
```

Comment:
it seems reasonable to define this, but it actually gets in our way for core plugins since we set the node version there for all users